8.x/theme-development/client-side-cart-module.md

```diff
@@ -0,0 +1,52 @@
+## Client Side cart module
+
+[[toc]]
+
+The Larammerce Template Engine has the **local card module**. In this tutorial, the source code of this module is Reviewed.
+
+**local card module** is in the following directory `resources/assets/js/define/local_cart_service.js`.
```

Comment:
**LocalCartService** is placed in the `...` file consisting of 437 lines of code. It is based on `require js` module. To read more ... 👍🏻